### DISCUSSION TITLE
What is Vercel's IPS tag for UK domains?

### DISCUSSION THREAD

**Discussion by michrome** [vercel/vercel#5591] [Help] [answered]
I'm trying to transfer in a .uk domain into Vercel. To start the process I need Vercel's IPS tag. Anyone know where I can find this? Thanks!

**Comment by michrome** [comment, marked as answer]
From Vercel Customer Support:

> Unfortunately, we don't allow transfer of `.co.uk` TLDs at this time given the information required by the TLD authority.

**Comment by Dashue** [comment]
@michrome 
What would be the best course of action if a transfer is not allowed/supported for .co.uk?
Is there a workaround, do we wait until it is supported, or do we give up?

**Reply by michrome** [reply]
Hey @Dashue the workaround is simply to leave your `.co.uk` domain with your existing registrar and update your DNS settings there. It works flawlessly. [Here’s the guide](https://vercel.com/docs/custom-domains#step-4:-configuring-the-domain).

In short:
* For **[A](https://vercel.com/docs/custom-domains#apex-domains)** records, use `76.76.21.21`
* For **[CNAME](https://vercel.com/docs/custom-domains#subdomains)** records, use `cname.vercel-dns.com`
* For SSL certificates, have your **[CAA](https://vercel.com/support/articles/domain-not-generating-ssl-certificate?query=Certifi#verifying-the-caa-record)** record use `0 issue "letsencrypt.org" `

**Comment by Dashue** [comment]
No I mean work around to have my domain on vercel. I don't want to just
redirect it, I want to pay and manage it through vercel

**Comment by alasdair009** [comment]
Is it possible yet to transfer and manage a .co.uk domain on Vercel yet? Would like to do this for my site.
